content/concept-transaction-cost.md

```diff
@@ -101,6 +102,7 @@ _Fee levels_ represent the proportional difference between the minimum cost and
 | Reference transaction (most transactions) | 10 | 256 | 20 | 512 |
 | [Multi-signed transaction](reference-transaction-format.html#multi-signing) with 4 signatures | 50 | 256 | 100 | 512 |
 | [Key reset transaction](concept-transaction-cost.html#key-reset-transaction) | 0 | (Effectively infinite) | N/A | (Effectively infinite) |
+| [EscrowFinish transaction](reference-transaction-format.html#escrowfinish) with 32-byte preimage. | 340 | 245 | 680 | 512 |
```

Comment:
Wait, at 512 fee units the tx cost goes to 360 drops? Not 700 drops? How does _that_ formula work?